### DISCUSSION TITLE
Cross compilation w/ native-image?

### DISCUSSION THREAD

**Discussion by perezd** [oracle/graal#4256] [Q&A] [answered]
Has anyone achieved this successfully using the llvm-toolchain functionality? I'm wondering if it's possible to cross compile a native image for MacOS from Linux, for example.

**Comment by alina-yur** [comment, marked as answer]
Hi @perezd, the cross-compilation work and current limitations are is discussed in this issue: https://github.com/oracle/graal/issues/407
One thing you could probably look at is using GitHub actions, if that's an option in your setup: https://recursive.codes/blog/post/1540
